### PR TITLE
migration: Add case about setmaxdowntime/getmaxdowntime

### PR DESCRIPTION
**Pull request by cliping**
VIRT-298169 - [migrate-s(g)etmaxdowntime] set/get vm max downtime

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

 (1/3) type_specific.io-github-autotest-libvirt.migration.migration_cmd.setmaxdowntime_and_getmaxdowntime.vm_running.downtime_100: STARTED
 (1/3) type_specific.io-github-autotest-libvirt.migration.migration_cmd.setmaxdowntime_and_getmaxdowntime.vm_running.downtime_100: PASS (90.42 s)
 (2/3) type_specific.io-github-autotest-libvirt.migration.migration_cmd.setmaxdowntime_and_getmaxdowntime.vm_running.downtime_0: STARTED
 (2/3) type_specific.io-github-autotest-libvirt.migration.migration_cmd.setmaxdowntime_and_getmaxdowntime.vm_running.downtime_0: PASS (91.34 s)
 (3/3) type_specific.io-github-autotest-libvirt.migration.migration_cmd.setmaxdowntime_and_getmaxdowntime.vm_shutoff: STARTED
 (3/3) type_specific.io-github-autotest-libvirt.migration.migration_cmd.setmaxdowntime_and_getmaxdowntime.vm_shutoff: PASS (55.18 s)

**Comment by cliping** [comment]
modular redesign

**Review by nanli1 — APPROVED** [review]

File: libvirt/tests/src/migration/migration_cmd/setmaxdowntime_and_getmaxdowntime.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        vm_status = params.get("vm_status")
+
+        test.log.info("Run test for migrate-setmaxdowntime/migrate-getmaxdowntime.")
+        if vm_status == "vm_running":
+            orig_maxdowntime = int(virsh.migrate_getmaxdowntime(vm_name).stdout.strip())
+            ret = virsh.migrate_setmaxdowntime(vm_name, downtime, debug=True)
+            if ret.exit_status:
+                if status_error and downtime == "0":
+                    libvirt.check_result(ret, error_msg)
+            new_maxdowntime = int(virsh.migrate_getmaxdowntime(vm_name).stdout.strip())
```
Comment (chunfuwen) [inline libvirt/tests/src/migration/migration_cmd/setmaxdowntime_and_getmaxdowntime.py:35]: stdout --> stdout_text

Comment (cliping) [reply libvirt/tests/src/migration/migration_cmd/setmaxdowntime_and_getmaxdowntime.py]: Updated, thanks.

**Review by chunfuwen — APPROVED** [review]
lgtm

**Pull request merged**